src/main/java/com/extendedclip/deluxemenus/requirement/BooleanRequirement.java

```diff
@@ -0,0 +1,29 @@
+package com.extendedclip.deluxemenus.requirement;
+
+import com.extendedclip.deluxemenus.menu.MenuHolder;
+
+public class BooleanRequirement extends Requirement {
+
+  private final String bool;
+  private final boolean invert;
+
+  public BooleanRequirement(String bool, boolean invert) {
+    this.bool = bool;
+    this.invert = invert;
+  }
+
+  @Override
+  public boolean evaluate(MenuHolder holder) {
+    String check = holder.setPlaceholdersAndArguments(bool);
+    try {
+      boolean bool = Boolean.parseBoolean(check);
```

Comment:
I am not 100% sure if we want to use this method of parsing a boolean. It might cause confusion since if the value is anything other than "true" (ignoring case), it is considered to be "false". Maybe, instead, we might want to change the requirement name to "is true" or something similar to that?